ts_src/confidential.ts

```diff
@@ -75,6 +83,27 @@ export function unblindOutput(
   };
 }
 
+export function unblindWitnessUtxo(
```

Comment:
This should be renamead unblindOutputWithKey like its [go-elements' counterpart](https://github.com/vulpemventures/go-elements/blob/master/confidential/confidential.go#L31)

Could you also add unblindOutputWithNonce at this point?